README.md

```diff
@@ -40,10 +40,15 @@ Feel free to follow the next action points:
 * Generate your private key and certificate request: `./easyvpn request <your-jenkins-username>`
   Your private key will be generated in `./cert/pki/private`
 
-  > ### This key **must** remain **secret**!
+  > ⚠️ This key **must** remain **secret**! ⚠️
+
+* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN
```

Comment:
this just sounds like double work, can't you just put the reason in the PR?

(PR template could be used)

we get so few VPN requests anyway that we will probably know why someone is requesting it